Cargo.toml

```diff
@@ -23,6 +23,9 @@ bitcoin_hashes = "0.10"
 flate2 = "1.0"
 tar = "0.4"
 
+[patch.crates-io]
```

Comment:
`core-rpc` version `0.15` has been released :tada: ! ready to remove this patch and mark PR ready to review.